src/index.css

```diff
@@ -0,0 +1,13 @@
+body {
```

Comment:
we can delete this too and add the global values * { margin: 0, padding: 0, box-sizing: border-box; }